### ISSUE TITLE
Enable extension only for XML-like languages

### ISSUE THREAD

**Issue by yurikus** [formulahendry/vscode-auto-close-tag#22]
I have a C/C++ workspace and auto close tag extension is triggered for #include statements: 
`#include <cstdio></cstdio>`

**Comment by formulahendry** [comment]
In VS Code, File->Preferences->User Settings, to open the settings.json. Then add config like below:
``` json
{
    "auto-close-tag.activationOnLanguage": [
        "javascript",
        "javascriptreact",
        "html",
        "xml",
        "php"
    ]
}
```
Refer to [this](https://code.visualstudio.com/docs/languages/identifiers) for the full list of Language Identifiers.

**Comment by Inrego** [comment]
imo this should be default behavior, with settings options to enable it in other languages.

**Comment by formulahendry** [comment]
@Inrego @yurikus Thanks for your feedback! I have plan to only enable it on certain languages: #26 . Look forward to your more suggestion! 😄

**Comment by formulahendry** [comment]
Resolved by #26
